hls4ml/templates/vivado/nnet_utils/nnet_activation_stream.h

```diff
@@ -331,6 +333,66 @@ void softmax(hls::stream<data_T> &data, hls::stream<res_T> &res){
     }    
 }
 
+// template <class data_T, class res_T, typename CONFIG_T>
```

Comment:
Commented out code, printouts, etc, needs to be removed. New printouts should also be removed.